drivers/gpio/gpio_xlnx_axi.c

```diff
@@ -248,75 +397,78 @@ static const struct gpio_driver_api gpio_xlnx_axi_driver_api = {
 	.get_pending_int = gpio_xlnx_axi_get_pending_int,
 };
 
-#define GPIO_XLNX_AXI_GPIO2_HAS_COMPAT_STATUS_OKAY(n)			\
```

Comment:
It's very difficult to tell what is being changed in the code below since your commit contains both functional and whitespace changes. Please separate the two or leave out the whitespace changes all together.

Reply:
I split the whitespace changes out into a separate commit.